src/test/java/com/commercetools/pingpong/model/GameTest.java

```diff
@@ -0,0 +1,312 @@
+package com.commercetools.pingpong.model;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.relaxng.datatype.DatatypeException;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.*;
+
+
+public class GameTest {
+
+
+
+    public Player getDummyOne(int setScore, int matchScore, boolean serve, boolean beginningServe) {
+        Player dummyPlayerOne = new Player();
+
+        for (dummyPlayerOne.getSetScore(); dummyPlayerOne.getSetScore() < setScore;) {
```

Comment:
What do you need the `for` loop for?

Reply:
When calling the dummy method, you must give it a setPoint number. This loop adds *setPoints* to the **player** until the player has the correct number of *setPoints*...

Reply:
I see. In this case we might need to make it simpler to add points. We can do that in a separate PR though. Maybe leave a comment to explain that as it wasn't that clear for me. Thanks 😉